src/templates/.eslintrc.ejs

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true
+  },
```

Comment:
why not just call the `npx eslint` after creating the project? so the developer could use the eslint CLI to generate it as they need/want to, we don't need to generate/manage it, just pass this job for eslint CLI

Reply:
Would it make sense to keep the `lint --fix` command as the responsibility of the `herbs update` command in this case?

Reply:
Only the `lint`, so we will warn the developer about the problems, run the `lint --fix` could be so much invasive for the developer, for some reason they may want to maintain the code with the "lint problems"